rpc_jobs/dep_update.yml

```diff
@@ -0,0 +1,51 @@
+- project:
+    name: "Dependency-Update-Jobs"
+    repo:
+      - rpc-openstack:
+          URL: "https://github.com/rcbops/rpc-openstack"
+          BRANCH: "master"
```

Comment:
How do we make sure that newton is also updated? Would this be the same project or a separate project? Would it be another ``repo`` in the list?

Reply:
Another repo in the list.